### PR TITLE
Swap F-1 and Cleanup

### PR DESCRIPTION
**Pull request by Capkirk123**
Replace the SSTU F-1 Model with the BDB F-1 model (its better proportioned, and has toggelable insulation)

Remove unused models/textures

Fix improperly configured files for PatchManager

Should resolve https://github.com/KSP-RO/ROEngines/issues/79
(Not tested)

### CODE REVIEW

File: GameData/ROEngines/PartConfigs/F1B_BDB.cfg [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	subcategory = 0
+	RSSROConfig = true
+	RP0conf = true
+
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
```
Comment (DRVeyl) [inline GameData/ROEngines/PartConfigs/F1B_BDB.cfg:17]: Can you make use of the tooling `ROESetEngineDefaults` established in #78 ?

File: GameData/ROEngines/PartConfigs/F1B_BDB.cfg [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+	tags = f1, f-1, f 1, f1b, f-1b. f 1b, SLS, pyrios
+
+	engineType = F1B
+	stagingIcon = LIQUID_ENGINE
+	stageOffset = 1
+	childStageOffset = 1
+
+	TechRequired = start
+	cost = 1
+	entryCost = 1
```
Comment (DRVeyl) [inline GameData/ROEngines/PartConfigs/F1B_BDB.cfg:52]: `ROESetEngineDefaults` established in #78 ?

File: GameData/ROEngines/PartConfigs/F1_BDB.cfg [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+	subcategory = 0
+	RSSROConfig = true
+	RP0conf = true
+	
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
```
Comment (DRVeyl) [inline GameData/ROEngines/PartConfigs/F1_BDB.cfg:17]: `ROESetEngineDefaults` established in #78 ?

File: GameData/ROEngines/PartConfigs/F1_BDB.cfg [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+	tags = apollo, saturn, saturn-v, s-ic, saturn-5, f1, f-1
+	
+	engineType = F1
+	stagingIcon = LIQUID_ENGINE
+	stageOffset = 1
+	childStageOffset = 1
+	
+	TechRequired = start
+	cost = 1
+	entryCost = 1
```
Comment (DRVeyl) [inline GameData/ROEngines/PartConfigs/F1_BDB.cfg:52]: `ROESetEngineDefaults` established in #78 ?

**Review by DRVeyl — CHANGES_REQUESTED** [review]
Everything else looks good, at least from a static review of the changes.

**Comment by Capkirk123** [comment]
Defaults set

**Comment by vader111** [comment]
@Capkirk123 you still have to remove all the values that the ROESetEngineDefaults sets

**Review by DRVeyl — APPROVED** [review]

**Pull request merged**